addons/io_scene_gltf2/blender/com/gltf2_blender_material_helpers.py

```diff
@@ -20,7 +20,30 @@ def get_gltf_node_name():
 def create_settings_group(name):
     gltf_node_group = bpy.data.node_groups.new(name, 'ShaderNodeTree')
     gltf_node_group.inputs.new("NodeSocketFloat", "Occlusion")
+    thicknessFactor  = gltf_node_group.inputs.new("NodeSocketFloat", "Thickness")
+    thicknessFactor.default_value = 1.0
     gltf_node_group.nodes.new('NodeGroupOutput')
     gltf_node_group_input = gltf_node_group.nodes.new('NodeGroupInput')
     gltf_node_group_input.location = -200, 0
-    return gltf_node_group
\ No newline at end of file
+    return gltf_node_group
+
+def get_gltf_pbr_non_converted_name():
+    return "glTF Non Converted BPR Extensions"
+
+def create_gltf_pbr_non_converted_group(name):
+    gltf_node_group = bpy.data.node_groups.new(name, 'ShaderNodeTree')
+
+    specularTexture = gltf_node_group.inputs.new("NodeSocketFloat", "specularTexture")
+    specularTexture.default_value = 1.0
+    specularColorTexture = gltf_node_group.inputs.new("NodeSocketColor", "specularColorTexture")
+    specularColorTexture.default_value = [1.0,1.0,1.0,1.0]
+
+    sheenColorTexture = gltf_node_group.inputs.new("NodeSocketColor", "sheenColorTexture")
+    sheenColorTexture.default_value = [0.0,0.0,0.0,1.0]
+    sheenRoughnessTexture = gltf_node_group.inputs.new("NodeSocketFloat", "sheenRoughnessTexture")
```

Comment:
For these inputs, don't say "Texture" on the end of the name.

- Specular
- Specular Color
- Sheen  (unless we're using Principled BSDF for this)
- Sheen Color
- Sheen Roughness

These are each available in glTF as both a "Factor" (for uniform values) and a "Texture" (for images).  As with many common inputs (base color, roughness, etc.) the exporter can automatically decide to use a factor when the input is unconnected, a texture when it's connected, or both when there's a multiply node in between.

Reply:
Yes, sure. This of course take factor and/or texture, depending on node tree, even if the name doesn't say it.
I renamed these socket
But current channel mapping implementation made me choose a name that is not already a socket name from Principled Shader.
I currently added "glTF" at end of names.
This can be discussed monday